src/dcos_e2e/cluster.py

```diff
@@ -360,13 +356,19 @@ def install_dcos_from_url(
             build_artifact: The URL string to a build artifact to install DC/OS
                 from.
             dcos_config: The contents of the DC/OS ``config.yaml``.
+            files_to_copy_to_genconf_dir: Pairs of host paths to paths on
+                the installer node. These are files to copy from the host to
+                the installer node before installing DC/OS.
             log_output_live: If `True`, log output of the installation live.
                 If `True`, stderr is merged into stdout in the return value.
         """
         try:
             self._cluster.install_dcos_from_url_with_bootstrap_node(
                 build_artifact=build_artifact,
                 dcos_config=dcos_config,
+                files_to_copy_to_genconf_dir=list(
```

Comment:
Why cast to a list?

Reply:
Random, was done before somewhere. I can remove it.